pkg/webui/console/views/application/index.js

```diff
@@ -169,17 +175,22 @@ export default class Application extends React.Component {
     }
 
     return (
-      <Switch>
-        <Route exact path={`${match.path}`} component={ApplicationOverview} />
-        <Route path={`${match.path}/general-settings`} component={ApplicationGeneralSettings} />
-        <Route path={`${match.path}/api-keys`} component={ApplicationApiKeys} />
-        <Route path={`${match.path}/link`} component={ApplicationLink} />
-        <Route path={`${match.path}/devices`} component={Devices} />
-        <Route path={`${match.path}/collaborators`} component={ApplicationCollaborators} />
-        <Route path={`${match.path}/data`} component={ApplicationData} />
-        <Route path={`${match.path}/payload-formatters`} component={ApplicationPayloadFormatters} />
-        <Route path={`${match.path}/integrations`} component={ApplicationIntegrations} />
-      </Switch>
+      <React.Fragment>
+        <IntlHelmet
+          titleTemplate={m.title} values={{ appId }}
```

Comment:
```suggestion
          titleTemplate={m.title} values={{ appId || appName }}
```
See my other comment